data/lib/src/datasource_impl/shared_space_document_datasource_impl.dart

```diff
@@ -93,4 +103,30 @@ class SharedSpaceDocumentDataSourceImpl implements SharedSpaceDocumentDataSource
 
     return FileUploadState(mergedStream, UploadTaskId(taskId));
   }
+
+  @override
+  Future<List<WorkGroupNode>> getAllChildNodes(
+    SharedSpaceId sharedSpaceId, [WorkGroupNodeId parentNodeId]) {
+    return Future.sync(() async {
+      return (await _linShareHttpClient.getWorkGroupChildNodes(sharedSpaceId, parentNodeId))
+          .map<WorkGroupNode>((workgroupNode) {
+            if (workgroupNode is WorkGroupDocumentDto) return workgroupNode.toWorkGroupDocument();
+
+            if (workgroupNode is WorkGroupNodeFolderDto) return workgroupNode.toWorkGroupFolder();
+
+            developer.log('Cannot cast workgroup node = $workgroupNode');
+            return null;
```

Comment:
I think we should avoid `null`, how about just return `workgroupNode`

Reply:
or a constant instead of `null`

Reply:
I tried another way is filter out node is null at the end. Plz help me review review at commit 7cf130e6464e61229948d264f50448e18a750a27